app/javascript/gobierto_data/webapp/components/Sidebar.vue

```diff
@@ -53,24 +54,31 @@ export default {
     filters: {
       type: Array,
       default: () => []
+    },
+    items: {
+      type: Array,
+      default: () => []
     }
   },
   data() {
     return {
       labelSets: I18n.t("gobierto_data.projects.sets") || "",
       labelQueries: I18n.t("gobierto_data.projects.queries") || "",
       labelCategories: I18n.t("gobierto_data.projects.categories") || "",
-      currentTabComponent: null,
-      currentTab: this.activeTab
+      currentTabComponent: null
     };
   },
+  watch: {
+    activeTab(newValue) {
+      this.currentTabComponent = COMPONENTS[newValue];
```

Comment:
Wrap these line with conditionals
```suggestion
    activeTab(newValue, oldValue) {
    if (newValue !== oldValue) {
          this.currentTabComponent = COMPONENTS[newValue];
    }
```
Always do so when you watch properties, otherwise this could be an infinite loop